### PR TITLE
add fim model controller migrations for ui

### PR DESCRIPTION
**Pull request by emilyville**
Add fim bits for UI
- start
- goal
- finish
- 90day
- 1year

& fim partial

### CODE REVIEW

File: app/models/domain/fim_measurements.rb [hunk 1]
```diff
@@ -1,3 +0,0 @@
-class Domain::FimMeasurements < ActiveRecord::Base
```
Comment (awong-dev) [inline app/models/domain/fim_measurements.rb]: It occurs to me that we should actually restore this file.  I think there is both a domain_fim_measurements and a fim_measurements table.

**Comment by awong-dev** [comment]
Gonna merge for now even though we should restore that file cause this is too big for me to want to leave sitting over the weekend.

**Comment by awong-dev** [comment]
LGTM

File: app/models/domain/fim_measurements.rb (hunk 1 above)
Comment (emilyville) [inline app/models/domain/fim_measurements.rb]: I will restore for the next PR!

**Pull request merged**